recipe/meta.yaml

```diff
@@ -23,24 +23,22 @@ requirements:
   build:
     - {{ compiler('cxx') }}
   host:
-    - python
+    - python !=3.11.0,!=3.11.1
```

Comment:
Is this ok here, or should it be included in `skip`?

Reply:
Setting this here overrides the cbc configuration. It should be in a run_constrained instead. (You cannot express avoiding minors with skip.)